### PR TITLE
auto redirect to init page if new instance

### PR DESCRIPTION
**Pull request by lwhiteley**
first attempt at doing an auto redirect if the relax instance is new

> not sure if its the right approach

wasnt sure about using the user-count query becase it was protected and also dont know the performance when there are many users.

So created a setting bit to see if the admin was setup. so any already created instances will need to add this setting and set to `true`

```json
{
    "_id" : "initialized",
    "__v" : 0,
    "value" : true
}
```

wanted you to review this before i did anything further.
@bruno12mota

### CODE REVIEW

**Comment by bruno12mota** [comment]
Nice one! Although using a Setting might not be the best approach because if you remove all users you need to be able to go to init. I think exposing the user counts query would be the best option, not critical that this is protected, not a critical piece of information.

**Comment by lwhiteley** [comment]
kk ill make the switch

**Comment by lwhiteley** [comment]
updated

File: lib/shared/routers/public.js [hunk 1]
```diff
@@ -3,22 +3,46 @@ import Public from 'screens/public';
 import React from 'react';
 import SchemaEntry from 'screens/public/screens/schema-entry';
 import {Route, IndexRoute} from 'react-router';
+import request from 'helpers/request';
+
+function initialized (nextState, replaceState, callback) {
```
Comment (bruno12mota) [inline lib/shared/routers/public.js]: This logic should be in the server side route https://github.com/relax/relax/blob/master/lib/server/routers/auth.js#L79 this way this route is never accessible.

Comment (bruno12mota) [inline lib/shared/routers/public.js]: Also no need to use this for the public routes. The change should only be for the /admin/login route

**Comment by lwhiteley** [comment]
updated

**Comment by bruno12mota** [comment]
Awesome 👍

**Pull request merged**